src/lengths.jl

```diff
@@ -23,4 +21,6 @@ function gyroradius(B, T::EnergyOrTemp; mass_numb=1, Z=1)
     return gyroradius(B, Vperp; mass_numb, Z)
 end
 
-gyroradius(val::Union{Velocity, EnergyOrTemp}, B::BField; kw...) = gyroradius(B, val; kw...)
\ No newline at end of file
+gyroradius(val::Union{Velocity, EnergyOrTemp}, B::BField; kw...) = gyroradius(B, val; kw...)
+
+const rc_ = gyroradius
```

Comment:
I don't really like defining this shorthand, because I wouldn't know that `rc_` was supposed to mean gyroradius if I saw it by itself in the wild.

Reply:
I add this only because `plasmapy` has it, and it makes easier to migrate from python to julia